CMakeLists.txt

```diff
@@ -123,7 +123,7 @@ option(WITH_OPENMP "Build with OpenMP support" ON)
 # Data format options
 option(WITH_TIFF "Build with LibTIFF support ." ON)
 option(WITH_NETCDF "Build with NetCDF support" OFF)
-option(WITH_GEOS "Build with GEOS support" OFF)
+option(WITH_GEOS "Build with GEOS support" ON)
```

Comment:
Is GEOS normally always available on the platforms we would be building for?

Would it be automatically disabled if not found or automatically enabled if found?

Reply:
> Is GEOS normally always available on the platforms we would be building for?

I would say yes.

> Would it be automatically disabled if not found or automatically enabled if found?

Currently nothing automatic (see https://github.com/OSGeo/grass/blob/main/cmake/modules/CheckDependentLibraries.cmake) , which is good from a packager point of view (no opportunistic linking).

Reply:
Ok, perfect!